<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_ui_message">
    <sys_ui_message action="INSERT_OR_UPDATE">
        <application/>
        <code/>
        <commented_message/>
        <key>encountered an unexpected value for the field "previous_cr_state". value: "{0}"</key>
        <language>uk</language>
        <message>Виявлено непередбачене значення для поля previous_cr_state. Значення: "{0}"</message>
        <sys_class_name>sys_ui_message</sys_class_name>
        <sys_created_by>bogdan.felinskyi</sys_created_by>
        <sys_created_on>2022-03-23 17:35:50</sys_created_on>
        <sys_id>276eb4a31b96c910c069657bab4bcbe7</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>encountered an unexpected value for the field "previous_cr_state". value: "{0}"</sys_name>
        <sys_package display_value="I18N: Ukrainian Translations" source="176ebda287b38910f4b131570cbb3551">176ebda287b38910f4b131570cbb3551</sys_package>
        <sys_policy/>
        <sys_scope display_value="I18N: Ukrainian Translations">176ebda287b38910f4b131570cbb3551</sys_scope>
        <sys_update_name>sys_ui_message_276eb4a31b96c910c069657bab4bcbe7</sys_update_name>
        <sys_updated_by>bogdan.felinskyi</sys_updated_by>
        <sys_updated_on>2022-03-23 17:35:50</sys_updated_on>
    </sys_ui_message>
</record_update>
